lib/containers/artist.js

```diff
@@ -54,6 +62,12 @@ export default Relay.createContainer(Artist, {
   fragments: {
     artist: () => Relay.QL`
       fragment on Artist {
+        partner_shows {
+          _id
+        }
```

Comment:
Do it, me and @ashfurrow  did - https://ashfurrow.com/blog/lets-steal-other-developers-best-ideas/